kong/tools/utils.lua

```diff
@@ -13,6 +13,7 @@ local uuid = require "resty.jit-uuid"
 local pl_stringx = require "pl.stringx"
 local pl_stringio = require "pl.stringio"
 local pl_utils = require "pl.utils"
+local nkeys = require "table.nkeys"
```

Comment:
`nkeys` is only used by `validate_labels`, so it should be moved into that `do end` block.